webpack/ForemanTasks/Routes/ForemanTasksRoutes.test.js

```diff
@@ -1,6 +1,5 @@
 import React from 'react';
-import { shallow } from 'enzyme';
-import toJson from 'enzyme-to-json';
+import toJson, { shallow } from '@theforeman/test';
```

Comment:
I see it odd that the default import is the `toJson`, would expect to do `import { shallow, enzymeToJson } from '@theforeman/test';`